### PR TITLE
COVERALL: Improve maintenance and readbility of some variables

### PR DESCRIPTION
**Pull request by vncoelho**

### CODE REVIEW

**Comment by Jim8y** [comment]
LGTM

**Review by Jim8y — DISMISSED** [review]

**Comment by shargon** [comment]
still not working

**Review by shargon — COMMENTED** [review]
I prefer to fix the issue instead of change the style (or both together)

**Comment by vncoelho** [comment]
> I prefer to fix the issue instead of change the style (or both together)

You can try the fix in another PR.
I still did not figure out the fix.

**Comment by Jim8y** [comment]
> I think there is some error introduced @Jim8y
> 
> ```shell
> [coverlet] MergeWith: file ''/home/runner/work/neo/neo/TestResults/coverage/coverage.json'' does not exist.
>    Generating report '/home/runner/work/neo/neo/tests/Neo.Plugins.Storage.Tests/'/home/runner/work/neo/neo/TestResults/coverage/'.info'
> ```
> 
> I just checked here on the report. Let me try to fix.

I compared the string,,,,, they are the same.... not sure whats wrong

**Review by shargon — CHANGES_REQUESTED** [review]
maybe we need different arguments in different calls, I think that this pr is not required, we need to solve the coverage issue, after it, maybe this change is not required.

**Comment by vncoelho** [comment]
> maybe we need different arguments in different calls, I think that this pr is not required, we need to solve the coverage issue first, after it, maybe this change is not required.

After the fix I will open this again

**Comment by vncoelho** [comment]
> > maybe we need different arguments in different calls, I think that this pr is not required, we need to solve the coverage issue first, after it, maybe this change is not required.
> 
> After the fix I will redo this again

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -40,18 +40,21 @@ jobs:
...
         sudo apt-get --assume-yes install libleveldb-dev librocksdb-dev
-        dotnet test ./tests/Neo.Cryptography.BLS12_381.Tests /p:CollectCoverage=true /p:CoverletOutput='${{ github.workspace }}/TestResults/coverage/'
-        dotnet test ./tests/Neo.ConsoleService.Tests /p:CollectCoverage=true /p:CoverletOutput='${{ github.workspace }}/TestResults/coverage/' /p:MergeWith='${{ github.workspace }}/TestResults/coverage/coverage.json'
-        dotnet test ./tests/Neo.UnitTests /p:CollectCoverage=true /p:CoverletOutput='${{ github.workspace }}/TestResults/coverage/' /p:MergeWith='${{ github.workspace }}/TestResults/coverage/coverage.json'
-        dotnet test ./tests/Neo.VM.Tests /p:CollectCoverage=true /p:CoverletOutput='${{ github.workspace }}/TestResults/coverage/' /p:MergeWith='${{ github.workspace }}/TestResults/coverage/coverage.json'
-        dotnet test ./tests/Neo.Json.UnitTests /p:CollectCoverage=true /p:CoverletOutput='${{ github.workspace }}/TestResults/coverage/' /p:MergeWith='${{ github.workspace }}/TestResults/coverage/coverage.json'
+        COVERALL_COLLECT_OUTPUT="/p:CollectCoverage=true /p:CoverletOutput='${{ github.workspace }}/TestResults/coverage/'"
```
Comment (cschuchardt88) [inline .github/workflows/main.yml:43]: Can you please use `github` environment to do this. Its easier to maintain and read/debug.

**Review by cschuchardt88 — CHANGES_REQUESTED** [review]

**Comment by vncoelho** [comment]
@cschuchardt88, sure! I will try with git-env.

I will reopen after the fix.

**Review by shargon — APPROVED** [review]

**Review by Jim8y — APPROVED** [review]

**Review by cschuchardt88 — APPROVED** [review]

**Pull request merged**